requirements.txt

```diff
@@ -1,2 +1,3 @@
 discord.py==2.1.0
 requests==2.28.1
+skforecast==0.6.0
```

Comment:
This adds time series forecasting on top of scikit-learn. Although scikit-learn has the regression models needed, this cuts out a lot of work needed to do time series forecasting like what match prediction needs. We're installing scikit-learn through the OS package manager because it's faster. This doesn't appear to be very big.